ddpui/tests/services/test_dbt_service.py

```
@@ -154,6 +195,760 @@ def run_dbt_init(*args, **kwargs):
+    # Mock warehouse connection
+    mock_warehouse = Mock()
+    mock_warehouse.get_table_columns.return_value = None  # Use manifest columns
+
+    with patch(
+        "ddpui.ddpdbt.dbt_service.WarehouseFactory.connect", return_value=mock_warehouse
+    ), patch(
+        "ddpui.ddpdbt.dbt_service.secretsmanager.retrieve_warehouse_credentials",
+        return_value={"host": "localhost"},
+    ):
+        # Parse the manifest - this should detect the operation chain and NOT create direct edge
+        result = parse_dbt_manifest_to_canvas(
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)